feature/visited/src/main/kotlin/uk/govuk/app/visited/ui/widget/VisitedWidget.kt

```diff
@@ -42,7 +42,12 @@ fun VisitedWidget(
         ) {
             Row(
                 modifier = Modifier
-                    .padding(GovUkTheme.spacing.large)
+                    .padding(
+                        start = GovUkTheme.spacing.medium,
+                        end = GovUkTheme.spacing.large,
```

Comment:
Should the start and end padding not match?

Reply:
Great spot @davidc-gds / @TimGJD - fixed up and updated screenshots